.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- The Dependabot team is currently at reduced capacity, because of this our
+response times on issues will be slower than we'd like. -->
+
+**Describe the bug**
+A clear and concise description of what the bug may be.
+
+**Package ecosystem**: <!-- bundler, docker, npm.. etc -->
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**PR, Issue or logs**
+If applicable, add links to public PR's or Issues that Dependabot opened, and/or
+paste in any related logs.
```

Comment:
I like this over what we have in the feedback repo. We currently ask for the repo but might be better to start asking for manifest files etc for easier reproduction instead.

Something like?
```
<!-- The more information you can provide, the easier it will be to reproduce the issue and find a fix -->

**Package manage/ecosystem**
<!-- ruby:bundler, javascript, docker.. etc -->
**Manifest contents prior to update**
<!-- If applicable, link to package.json, Gemfile.. etc -->
**Updated dependency**
<!-- If applicable, the dependency name and to and from versions -->
**What you expected to see, versus what you actually saw**
<!-- A clear and concise description of what you expected to happen -->
**Images of the diff or a link to the PR, issue or logs**
<!-- If applicable, add links to public PR's or Issues that Dependabot opened, and/or
paste in any related logs -->
```

Reply:
Yeah, I like that 👍